### DISCUSSION TITLE
方案选单中的「繁体->简体」，是否有类似 flag 值读取？

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1228] [Q&A]
咨询一下，例如我有一个方案 `luna_pinyin` ，它默认是繁体输出，我想不经过方案选单就设置为简体，是否有办法可以达到？

**Comment by LEOYoon-Tsaw** [comment]
可以設置快捷鍵，bindings

**Reply by pdog18** [reply]
谢谢哥，但是设置快捷键依旧需要手动按一下快捷键切换，rime 从来没有考虑设置这样一个开关么？

**Reply by LEOYoon-Tsaw** [reply]
>  依旧需要手动按一下快捷键切换

敢問您是想用意念操控？

**Reply by pdog18** [reply]
> > 依旧需要手动按一下快捷键切换
> 
> 敢問您是想用意念操控？

抱歉！

主要我想达到设置了方案就默认是简体拼音，例如方案「袖珍拼音」一样。

因为我想让用家在我制作的 UI 配置界面上，进行一通设置以后，输入法不需要任何额外的操作，就可以愉快的打字了。

假如，仍然需要 F4 来切换大小写，UI 配置界面就显得很弱。

**Reply by LEOYoon-Tsaw** [reply]
默認簡體可以設置reset :1
```yaml
switches:
  - name: simplification
    reset: 1
```

**Reply by pdog18** [reply]
> 默認簡體可以設置reset :1
> 
> ```yaml
> switches:
>   - name: simplification
>     reset: 1
> ```

感谢！
